Gordon360/Properties/launchSettings.json

```diff
@@ -3,12 +3,12 @@
     "Development": {
       "commandName": "Project",
       "launchBrowser": true,
-      "launchUrl": "https://localhost:5100/swagger",
+      "launchUrl": "https://localhost:51657/swagger",
```

Comment:
@RossClark01 I'm guessing we should revert this change, I assume this was committed by accident.

Reply:
These changes are fine, in the sense that the settings here are arbitrary and should be changed by every developer.